### PR TITLE
Update diagrams and note for hours and remark in Member and Event

### PR DESCRIPTION
**Pull request by ph-nathan**
What have you done in this PR?
* Cry 

Dev Testing Steps / Screenshots
image
image
image

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -199,11 +203,16 @@ solutions.
 This section explains how the `Event` model is implemented and the various design consideration when
 implementing this model.
 
-The class diagram below shows the current implementation of `Event` model.
+The `Event` and `UniqueEventList` classes are implemented as shown in the diagram below:
 
 ![EventClassDiagram](images/EventClassDiagram.png)
 
 Every `Event` contains a `Name`, `Location`, `EventDate`, `Optional<Hours>`, `Optional<Remark>` and a set of `Tags`
```
Comment (ph-nathan) [inline docs/DeveloperGuide.md]:
```suggestion
Every `Event` contains a `Name`, `Location`, `EventDate`, `Optional<Hours>`, `Optional<Remark>` and a set of `Tags`.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -164,8 +164,12 @@ implementing this model.
 
 The class diagram below shows the current implementation of `Member` model.
 
-![MemberClassDiagram](images/MemberClassDiagram.png)
-Every `Member` contains a `Name`, `Gender`, `Optional<Phone>`, `Optional<Email>`, `Optional<Address>`, `Optional<Hours>`, `Optional<Remark>` and a set of `Tags`
+<img src="images/MemberClassDiagram.png" width="500px" style="margin-left:120px">
+
+Every `Member` contains a `Name`, `Gender`, `Optional<Phone>`, `Optional<Email>`, `Optional<Address>`, `Optional<Hours>`, `Optional<Remark>` and a set of `Tags`. `UniqueMemberList` stores all unique instances of `Member`.
+
+Note that the `Hours` and a `Remark` fields will only be meaningfully intialised
```
Comment (dylkaw) [inline docs/DeveloperGuide.md]:
```suggestion
Note that the `Hours` and a `Remark` fields will only be meaningfully initialised
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -199,11 +203,16 @@ solutions.
...
-The class diagram below shows the current implementation of `Event` model.
+The `Event` and `UniqueEventList` classes are implemented as shown in the diagram below:
+
+<img src="images/EventClassDiagram.png" width="500px" style="margin-left:120px">
+
+Every `Event` contains a `Name`, `Location`, `EventDate`, `Optional<Hours>`, `Optional<Remark>` and a set of `Tags`.
+`UniqueEventList` stores all unique instances of `Event`.
 
-![EventClassDiagram](images/EventClassDiagram.png)
+Note that similar to `Member` class , the `Hours` and a `Remark` fields will only be meaningfully intialised
```
Comment (dylkaw) [inline docs/DeveloperGuide.md]:
```suggestion
Note that similar to `Member` class , the `Hours` and a `Remark` fields will only be meaningfully initialised
```

**Review by dylkaw — APPROVED** [review]
LGTM! Just two spelling errors.

**Review by wr1159 — APPROVED** [review]
LGTM

**Pull request merged**